user-manual/archival-storage/archival-storage.rst

```diff
@@ -49,6 +51,71 @@ documentation.
+
+.. image:: images/archival-storage-select-columns.*
+   :align: center
+   :width: 80%
+   :alt: The Select Columns options on the Archival Storage tab
+
+Click on a blue or white row to toggle this value for each column. In the example below, all the available columns are selected. Note that your column selection choices will persist between your user sessions and apply to other users of the same Archivematica pipeline.
+
+.. image:: images/archival-storage-all-columns.*
+   :align: center
+   :width: 80%
+   :alt: The Archival Storage tab with all columns selected
+
+You can click on the column headers to sort in ascending or descending order. An up or down caret (arrow point) will appear. (e.g. see **File count** below).
+
+.. image:: images/archival-storage-sorted-column.*
+   :align: center
+   :width: 80%
+   :alt: The Archival Storage tab sorted on File Count
+
+When you enable the **Show files?** option at the top of the screen, all of the files contained in Archival Storage AIPs are listed in the Browse archival storage table.
+
+.. image:: images/archival-storage-show-files.*
+   :align: center
+   :width: 80%
+   :alt: The Archival Storage tab with Show Files selected
+
+The **Select columns** option is also available on the **Show files?** table.
+
+.. image:: images/archival-storage-show-files-columns.*
+   :align: center
+   :width: 80%
+   :alt: Selecting columns for the Show Files view
+
+Column sorting is also available in the **Show files?** table (e.g. see **AIP** column below).
+
+.. image:: images/archival-storage-show-files-sort.*
+   :align: center
+   :width: 80%
+   :alt: Sorting Show Files view by AIP column
+
+Note also that the **AIP** column cells include a hyperlink to the AIP information page (e.g. "transfer1"). From that page it is possible to select an individual AIP and download it, delete it, or initiate AIP re-ingest.
+
+.. _csv-download:
+
+Downloading all AIP metadata in a CSV file
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+All the values displayed in all of the columns available on the Browse Archival Storage tab can be downloaded to a CSV file. This CSV file can be used for more detailed analysis and filtering of AIP information, beyond the Search capabilities described below. The CSV data can also be used as input to a reporting or charting tool.
+
+Click on the **Download CSV** link at the top right of the **Browse archival storage** table.
+
+.. image:: images/archival-storage-csv-download.*
+   :align: center
+   :width: 80%
+   :alt: Download all AIP metadata using CSV download
+
+You will be prompted for a place to store the CSV file. The CSV data will include values for all the columns, regardless of which ones are currently selected. It will also include the metadata for each AIP in archival storage, regardless of which ten are currently being displayed on the Archival Storage tab.
```

Comment:
> regardless of which ten are currently being displayed on the Archival Storage tab. 

Is quite specific. If we change this to something like:

> regardless of those that are currently displayed on the Archival Storage tab.

It might mean one less update here if, in future, we take advantage of the Data Tables features to make it configurable how many rows can be rendered at a time.

Reply:
Yeah, I paused on that one and reasoned that there is no way to change the 10 default ATM so went with that but I agree with your suggestion to future-proof it.